vNext/test/NonSiloTests/Orleans.NonSiloTests.csproj

```diff
@@ -42,6 +42,12 @@
     <Compile Include="..\..\..\test\NonSiloTests\SchedulerTests\OrleansTaskSchedulerAdvancedTests.cs">
       <Link>SchedulerTests\OrleansTaskSchedulerAdvancedTests.cs</Link>
     </Compile>
+    <Compile Include="..\..\..\test\TesterInternal\Serialization\IlBasedSerializerTests.cs">
```

Comment:
I moved the remaining tests in Serialization folder. And `BuiltInSerializerTests ` too since it's a dependency.

 I will deal with the merge conflicts, just a heads up that I moved them ~